editor/src/components/canvas/ui-jsx-canvas-renderer/ui-jsx-canvas-component-renderer.tsx

```diff
@@ -41,8 +41,9 @@ export type ComponentRendererComponent = React.ComponentType<{
   [UTOPIA_INSTANCE_PATH]: ElementPath
   [UTOPIA_PATHS_KEY]?: string
 }> & {
-  topLevelElementName: string
+  topLevelElementName: string | null
```

Comment:
It looks like `topLevelElementName` can be removed from here now (and from `Component.topLevelElementName`, since it doesn't appear to be used for anything after this work

Reply:
hm, I liked your comment at first, but I think it's "free" to have this here, and might make debugging easier

Reply:
Ok that's fair